src/test/scala/com/fortysevendeg/translatebubble/modules/clipboard/ClipboardServiceComponentSpec.scala

```diff
@@ -59,6 +59,55 @@ class ClipboardServiceComponentSpec
 
     }
 
+    "ClipboardService should validate text if the text value is not a number" in new ClipboardMocks {
+      val text = "This is not a number"
+      mockClipItem.getText returns text
+
+      clipboardServices.isValidCall must beTrue
+    }
+
+    "ClipboardService should not validate text if the text value is a number" in new ClipboardMocks {
+      val number = "47"
+      mockClipItem.getText returns number
+
+      clipboardServices.isValidCall must beFalse
+    }
+
+    "ClipboardService should not validate text if the text value is a URL starting with 'https'" in new ClipboardMocks {
+      val url = "https://www.47deg.com"
+      mockClipItem.getText returns url
+
+      clipboardServices.isValidCall must beFalse
+    }
+
+    "ClipboardService should not validate text if the text value is a URL starting with 'http'" in new ClipboardMocks {
+      val url = "http://www.47deg.com"
+      mockClipItem.getText returns url
+
+      clipboardServices.isValidCall must beFalse
+    }
+
+    "ClipboardService should not validate text if the text value is a URL starting with 'www'" in new ClipboardMocks {
+      val url = "www.47deg.com"
+      mockClipItem.getText returns url
+
+      clipboardServices.isValidCall must beFalse
+    }
+
+    "ClipboardService should not validate text if the text value is a URL starting with the domain" in new ClipboardMocks {
+      val url = "47deg.com"
+      mockClipItem.getText returns url
+
+      clipboardServices.isValidCall must beFalse
+    }
+
+    "ClipboardService should validate text if the text value is not a URL" in new ClipboardMocks {
```

Comment:
I'm sorry but I think that my explanation wasn't very clear. I was referring to remove one of them and change the description of the another one to `ClipboardService should validate text if the text value is not a number neither an url` since the code of both unit test is identical. But nevermind, we could leave the two unit tests too ;)